Gruvo/Gruvo/ClientApp/src/app/app.module.ts

```diff
@@ -21,7 +22,8 @@ import { AuthGuard } from '@app/auth.guard';
 import { FeedComponent } from '@app/feed/feed.component';
 import { FeedService } from '@app/feed/feed.service';
 import { RecommendationComponent } from '@app/recommendation/recommendation.component';
-
+import { CreateTweetComponent } from '@app/createtweet/createtweet.component';
```

Comment:
createtweet directory and createtweet.component. everything is lowercase, look https://angular.io/guide/styleguide#separate-file-names-with-dots-and-dashes